backend/sirius-web-diagrams-layout/src/main/java/org/eclipse/sirius/web/diagrams/layout/incremental/provider/EdgeLabelPositionProvider.java

```diff
@@ -25,7 +29,20 @@
  */
 public class EdgeLabelPositionProvider {
 
+    private LayoutConfigurator layoutConfigurator;
+
+    public void setLayoutConfigurator(LayoutConfigurator layoutConfigurator) {
+        this.layoutConfigurator = layoutConfigurator;
+    }
+
     public Position getCenterPosition(EdgeLayoutData edge, LabelLayoutData label) {
+        Double spacingEdgeLabel = null;
+        if (this.layoutConfigurator instanceof SiriusWebLayoutConfigurator) {
+            spacingEdgeLabel = ((SiriusWebLayoutConfigurator) this.layoutConfigurator).configure(ElkEdge.class).getProperty(CoreOptions.SPACING_EDGE_LABEL);
+        }
```

Comment:
Why does this need to be casted to SiriusWebLayoutConfigurator?

Reply:
Done